packages/tools/devtools/devtools-core/src/DevtoolsLogger.ts

```diff
@@ -43,7 +43,7 @@ import { type ITimestampedTelemetryEvent } from "./TelemetryMetadata";
  * TODO: Document others as they are added.
  *
  * @sealed
- * @internal
+ * @alpha
  */
```

Comment:
it a bit strange to expose this class. why is it necessary?

Reply:
if a consumer need to instantiate this, it could be better to follow the pattern we now follow in the fluid-static stuff to just expose a create function that returns an interface.

Reply:
Yeah, it violates our design principles. We should change this. I'll put up a separate PR to migrate to an interface + factory function pattern for this.

Reply:
Hmm I seem to remember we needed to know that we were being passed this, and not just any `ITelemetryBaseLogger` or _some_ object that implements an `IDevtoolsLogger` interface... we need the object we know has the necessary internal functionality to make the use case work. I wouldn't object too strongly to exposing n `IDevtoolsLogger` instead, taking that in the `FluidDevtoolsProps` and document that it's not supposed to be implemented by the users; just give us back what you got from the factory/free-function that gave you an `IDevtoolsLogger` in the first place.

Reply:
Documenting the interface to make clear we don't support external implementations is still probably an improvement over exporting the class, since it allows us to make (some) implementation detail changes without breaking users.